packages/orchestration/src/exos/chain-hub.js

```diff
@@ -108,6 +103,31 @@ export const makeChainHub = (agoricNames, zone = makeHeapZone()) => {
     valueShape: IBCConnectionInfoShape,
   });
 
+  // eslint-disable-next-line no-restricted-syntax -- TODO more exact rules for vow best practices
+  const lookupChainInfo = async chainName => {
+    await null;
+    try {
+      const chainInfo = await E(agoricNames).lookup(CHAIN_KEY, chainName);
+      chainInfos.init(chainName, chainInfo);
```

Comment:
Is there no risk of race with `registerChain` doing an `init` of the same name?

Reply:
thanks